pom.xml

```diff
@@ -54,6 +54,9 @@
         <release.skipTests>false</release.skipTests>
         <tagNameFormat>v@{project.version}</tagNameFormat>
         <useBeta>true</useBeta> <!-- For Jenkins.MANAGE permission -->
+        <!-- TODO JENKINS-73339 until in parent POM -->
+        <jenkins-test-harness.version>2250.v03a_1295b_0a_30</jenkins-test-harness.version>
+        <maven.compiler.testRelease>17</maven.compiler.testRelease>
```

Comment:
Temporary change to run tests with Jetty 12 (which requires Java 17). Note that PCT (Plugin Compatibility Tester) automatically applies these changes when this plugin is tested in `jenkinsci/bom` against a Jenkins core that contains Jetty 12 (e.g., the 2.472 pre-release, or the 2.472 weekly release that is scheduled for next Tuesday, August 13).